stats/config/charts.json

```diff
@@ -65,96 +65,96 @@
             "units": "Bytes"
         },
         "average_block_rewards": {
             "title": "Average block rewards",
-            "description": "Average amount of distributed reward in tokens per day",
+            "description": "Average amount of distributed rewards in tokens per period",
             "units": "{{native_coin_symbol}}"
         },
         "new_native_coin_transfers": {
-            "title": "New {{native_coin_symbol}} transfers",
-            "description": "New token transfers number for the period"
+            "title": "Number of {{native_coin_symbol}} transfers",
+            "description": "Token transfers performed during the period"
         },
         "native_coin_holders_growth": {
             "enabled": false,
             "title": "{{native_coin_symbol}} holders growth",
-            "description": "Cumulative token holders number for the period"
+            "description": "Cumulative number of token holders"
         },
         "new_native_coin_holders": {
             "enabled": false,
             "title": "New {{native_coin_symbol}} holders",
-            "description": "New token holders number per day"
+            "description": "New token holders number per period"
         },
         "native_coin_supply": {
             "enabled": false,
             "title": "{{native_coin_symbol}} circulating supply",
-            "description": "Amount of token circulating supply for the period",
+            "description": "Amount of token circulating supply per period",
             "units": "{{native_coin_symbol}}"
         },
         "average_gas_limit": {
             "title": "Average gas limit",
-            "description": "Average gas limit per block for the period"
+            "description": "Average block gas limit per period"
         },
         "gas_used_growth": {
             "title": "Gas used growth",
-            "description": "Cumulative gas used for the period"
+            "description": "Cumulative gas used"
         },
         "average_gas_price": {
             "title": "Average gas price",
-            "description": "Average gas price for the period (Gwei)",
+            "description": "Average gas price per period",
             "units": "Gwei"
         },
         "new_verified_contracts": {
             "title": "New verified contracts",
-            "description": "New verified contracts number for the period"
+            "description": "New verified contracts number per period"
         },
         "verified_contracts_growth": {
             "title": "Verified contracts growth",
-            "description": "Cumulative number verified contracts for the period"
+            "description": "Cumulative number of verified contracts"
         },
         "new_contracts": {
             "title": "New contracts",
-            "description": "New contracts number for the period"
+            "description": "New contracts number per period"
```

Comment:
Number of new contracts (verified and unverified)

Note: I believe this is correct, unless it is only unverified contracts.

Reply:
The verification is not relevant to this metric. It's number of contracts added to the network, and the verification happens off-chain.
With this in mind, "Number of new contracts in the network" (or in the chain) would clarify this, do you agree?

Reply:
yes, that should work.